third_party/move/move-core/types/src/value.rs

```diff
@@ -114,6 +114,10 @@ pub enum MoveTypeLayout {
     U32,
     #[serde(rename(serialize = "u256", deserialize = "u256"))]
     U256,
+
+    // Special marker which allows annotating aggregatable types.
+    // In the future, we might want to support different markings.
```

Comment:
So that would be one way of already introducing a generic enum type, internally providing a way to add other uses.
Alternatively, we could name it as AggregatorLifting or smt.